sys/xtimer/Kconfig

```
@@ -7,18 +7,44 @@
 
 menuconfig MODULE_XTIMER
     bool "xtimer"
+    depends on TEST_KCONFIG
+    help
+        Include xtimer module. xtimer requires a low-level timer implementation
+        that can be provided either by a peripheral timer or the ztimer module.
+
+if MODULE_XTIMER
+
+choice XTIMER_BACKEND
+    bool "xtimer backend"
+
+config MODULE_ZTIMER_XTIMER_COMPAT
+    bool "xtimer api implemented by ztimer"
+    select MODULE_DIV
+    select ZTIMER_USEC
+
+config MODULE_XTIMER_ON_PERIPH_TIMER
+    bool "xtimer api implemented by xtimer, with low level timer backend"
     depends on HAS_PERIPH_TIMER
     depends on TEST_KCONFIG
-    # use timer peripheral unless ztimer compatibility module is used
-    select MODULE_PERIPH_TIMER if HAS_PERIPH_TIMER && !MODULE_XTIMER_ON_ZTIMER && !MODULE_ZTIMER_XTIMER_COMPAT && !MODULE_ZTIMER64_XTIMER_COMPAT
+    select MODULE_PERIPH_TIMER if HAS_PERIPH_TIMER
```

Just realized here this also depends on:
```suggestion
    depends on !MODULE_ZTIMER_PERIPH_TIMER
    select MODULE_PERIPH_TIMER if HAS_PERIPH_TIMER
```